src/org/zaproxy/zap/extension/dynssl/ExtensionDynSSL.java

```diff
@@ -174,6 +196,103 @@ public X509Certificate getRootCaCertificate() {
 		}
 	}
 	
+	/**
+	 * Writes the Root CA public certificate to the specified file in pem format, suitable for importing into browsers
+	 * @param f the file the Root CA certificate will be written to
+	 * @throws IOException
+	 * @throws KeyStoreException
+	 */
+	public void writeRootPubCaCertificateToFile(File f) throws IOException, KeyStoreException {

```

Comment:
The new public methods could have the since tag, I'd also suggest using `Path` instead of `File`.